leftwm/src/bin/leftwm.rs

```diff
@@ -196,3 +211,22 @@ fn get_sigchld_flag() -> Arc<AtomicBool> {
 fn is_suspending(flag: &Arc<AtomicBool>) -> bool {
     !flag.swap(false, Ordering::SeqCst)
 }
+
+fn evaluate_session(leftwm_session: &mut Child) -> SessionStatus {
+    if let Ok(exit_status) = leftwm_session.wait() {
+        if exit_status.success() {
+            SessionStatus::Success
+        } else {
+            SessionStatus::Error
+        }
+    } else {
+        SessionStatus::DidNotStart
+    }
```

Comment:
Can this be refactored in a match case with `Ok() if ..`
How are we acting on `DidNotStart` anyways? We maybe want to add some commentary/doc about the StessionStatus and its implications.